src/interface/blas2/backend/amd_gpu.hpp

```diff
@@ -86,6 +86,22 @@ typename SB_Handle::event_t inline _sbmv(SB_Handle& sb_handle, index_t _N,
 }  // namespace backend
 }  // namespace sbmv
 
+namespace spmv {
+namespace backend {
+template <uplo_type uplo, typename SB_Handle, typename index_t,
+          typename element_t, typename container_t0, typename container_t1,
+          typename increment_t, typename container_t2>
+typename SB_Handle::event_t inline _spmv(SB_Handle& sb_handle, index_t _N,
```

Comment:
Maybe we can use `SYCL_BLAS_INLINE` here (and in the other places) to match with the changes inside `operations/blas2/spmv.hpp` file.